CDP4Reporting.Tests/CDP4Reporting.Tests.csproj

```diff
@@ -0,0 +1,76 @@
+    <OutputPath>bin\$(Platform)\$(Configuration)\</OutputPath>
+    <DebugType>pdbonly</DebugType>
+    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Castle.Core" Version="4.4.0" />
+    <PackageReference Include="CDP4Common-CE" Version="6.1.1" />
+    <PackageReference Include="CDP4Dal-CE" Version="6.1.0" />
+    <PackageReference Include="CDP4JsonSerializer-CE" Version="6.1.0" />
+    <PackageReference Include="CommonServiceLocator" Version="1.3" />
+    <PackageReference Include="DevExpress.Reporting.Core" Version="20.1.3" />
+    <PackageReference Include="DevExpress.Wpf.Reporting" Version="20.1.3" />
+    <PackageReference Include="Moq" Version="4.11.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="NLog" Version="4.6.8" />
+    <PackageReference Include="NUnit" Version="3.12.0" />
+    <PackageReference Include="Prism" Version="5.0.0" />
+    <PackageReference Include="Prism.Composition" Version="5.0.0" />
+    <PackageReference Include="Prism.Interactivity" Version="5.0.0" />
+    <PackageReference Include="Prism.MEFExtensions" Version="5.0.0" />
+    <PackageReference Include="Prism.Mvvm" Version="1.1.1" />
+    <PackageReference Include="Prism.PubSubEvents" Version="1.1.2" />
+    <PackageReference Include="reactiveui" Version="6.4.0" />
+    <PackageReference Include="reactiveui-core" Version="6.4.0" />
+    <PackageReference Include="Rx-Core" Version="2.2.5" />
+    <PackageReference Include="Rx-Interfaces" Version="2.2.5" />
+    <PackageReference Include="Rx-Linq" Version="2.2.5" />
+    <PackageReference Include="Rx-Main" Version="2.2.5" />
+    <PackageReference Include="Rx-PlatformServices" Version="2.2.5" />
+    <PackageReference Include="Rx-XAML" Version="2.2.5" />
+    <PackageReference Include="Splat" Version="1.6.0" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.7.0" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.2" />
+    <PackageReference Include="System.Xml.XmlSerializer" Version="4.3.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="PresentationCore" />
+    <Reference Include="PresentationFramework" />
+    <Reference Include="System.ComponentModel.Composition" />
+    <Reference Include="System.Configuration" />
+    <Reference Include="System.IO.Compression" />
+    <Reference Include="System.ServiceModel" />
+    <Reference Include="System.Transactions" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="WindowsBase" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\CDP4Composition\CDP4Composition.csproj" />
+    <ProjectReference Include="..\CDP4Reporting\CDP4Reporting.csproj" />
+  </ItemGroup>
+</Project>
```

Comment:
Some files are missing newlines at end of file.

Reply:
Yes, but I thing this rule is applied only to the source code files.

Reply:
@adrianchivu @cozminvelciu That is correct.